service/src/test/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatchTest.java

```diff
@@ -320,16 +357,19 @@ private void setupLandingZoneResources() {
                 .resourceId("Id31")
                 .resourceType("fooType31")
                 .region("fooRegion1")
+                .tags(Map.of("resourceTag1", "resourceValue1", "resourceTag2", "resourceValue2"))
                 .build(),
             LandingZoneResource.builder()
                 .resourceId("Id32")
                 .resourceType("fooType32")
                 .region("fooRegion2")
+                .tags(Map.of("resourceTag1", "resourceValue1", "resourceTag3", "resourceValue4"))
                 .build(),
             LandingZoneResource.builder()
                 .resourceId("Id33")
                 .resourceType("fooType33")
                 .region("fooRegion1")
+                .tags(Map.of("resourceTag1", "resourceValue1", "resourceTag3", "resourceValue4"))
```

Comment:
Just pattern matching here, but do you want this to be `resourceTag4` instead of `3`?